projects/igniteui-angular/src/lib/directives/scroll-inertia/scroll_inertia.directive.ts

```diff
@@ -329,6 +329,7 @@ export class IgxScrollInertiaDirective implements OnInit, OnDestroy {
         }
     }
 
+    // TODO remove this if it only supports IE?
```

Comment:
@skrustev Can you look at this code and remove it if it's IE11 specific?

Reply:
I've created a separate PR #10423 since it is not related to the tests.

Reply:
Thanks @skrustev! I will merge it after the build passes!